docs/what.rst

```diff
@@ -0,0 +1,23 @@
+=====================
+What is a Certificate?
+=====================
+
+An SSL *certificate* contains a public key and a private key that work together to enable
```

Comment:
1.  `s^is made up^consists^`

2. I'd say that the language is a bit better in this article: https://en.wikipedia.org/wiki/Public_key_certificate (compared to the article you referenced).

3. If I may:
_A public key certificate, or a digital certificate is a combination of a public key and a private key. It is used to be called "SSL certificate". The ultimate practical purpose of a certificate is to enable secure communication between a client program (web browser, e-mail client, etc.) and a server over an encrypted SSL/TLS (transport layer security) connection. The certificate is used to encrypt such a communication and to identify the computer (device)._

And that can be continued:
_The certificate includes information about the key, information about the identity of the computer and the digital signature of the issuer. If the issuer is trusted by the software initiating the communication and the signature is valid, the key can be used to communicate securely with the server identified by the certificate._

---
Note: Since the term "SSL certificate" is still being widely used, I think it would help people making the connection, if they've heard that term.